runtime/android/core/src/org/xwalk/core/XWalkActivityDelegate.java

```diff
@@ -66,29 +144,33 @@ public void onResume() {
 
     @Override
     public void onDecompressStarted() {
-        mDialogManager.showDecompressProgress(new Runnable() {
```

Comment:
If the library is not compressed, `onDecompressStarted()` won't be triggered, `DecompressTask` will jump to `onDecompressComplete` directly. So no need to modify this method because of download mode.